website/data/alert-banner.js

```diff
@@ -1,13 +1,13 @@
-export const ALERT_BANNER_ACTIVE = false
+export const ALERT_BANNER_ACTIVE = true
 
 // https://github.com/hashicorp/web-components/tree/master/packages/alert-banner
 export default {
-  tag: 'Blog post',
-  url: 'https://www.hashicorp.com/blog/a-new-chapter-for-hashicorp',
+  tag: 'New',
+  url: 'https://cloud.hashicorp.com/products/packer ',
   text:
-    'HashiCorp shares have begun trading on the Nasdaq. Read the blog from our founders, Mitchell Hashimoto and Armon Dadgar.',
-  linkText: 'Read the post',
+    'HCP Packer offers automation and security workflows for Packer, and is now generally available.',
+  linkText: 'Sign up for free',
   // Set the expirationDate prop with a datetime string (e.g. '2020-01-31T12:00:00-07:00')
   // if you'd like the component to stop showing at or after a certain date
-  expirationDate: '2021-12-17T23:00:00-07:00',
+  // expirationDate: '2021-12-17T23:00:00-07:00',
```

Comment:
Wondered if maybe we could see about an expiry in case this goes stale. Maybe there is another announcement planned  before that's a real concern?

Reply:
Yeah me too. I am awaiting a response on this :)